server/routes/routers/follows.ts

```diff
@@ -37,7 +37,12 @@ follow.get('/following/:id', async (req: any, res: any) => {
   try {
     const userFollowing = await prisma.user.findUnique({
       where: { id: +id },
-      include: { following: true },
+      select: {
+        id: true, 
+        username: true, 
+        picture: true,
+        following: true
```

Comment:
You cam nest a select to reduce the amount of data youre getting for each user like this: `following: {select: {id: true}},`